Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -200,7 +233,27 @@ public struct TLSConfiguration {
 
     /// The pre-TLS1.3 cipher suites supported by this handler. This uses the OpenSSL cipher string format.
     /// TLS 1.3 cipher suites cannot be configured.
-    public var cipherSuites: String
+    internal var cipherSuiteFormattedString: String = defaultCipherSuites
+    
+    /// Public property used to set the internal cipherSuiteFormattedString from NIOTLSCipher.
+    public var cipherSuite: [NIOTLSCipher] {
+        get {
+            // Default to a ECDHE_RSA based cipher.
+            return [.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256]
```

Comment:
This should presumably return the parsed version of `cipherSuiteFormattedString`?

Reply:
I removed the default `TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256` suite here and built an array of suites based on  what is contained in the `defaultCipherSuites` string (which is ultimately set to the cipherSuites property).  I felt like this put `cipherSuite` inline with `defaultCipherSuites`.  However, to do that I had to create a temporary SSL_CTX, set the suites, and then call into the `STACK(SSL_CIPHER)`'s  to extract the `uint16_t` value to match it against the same in `NIOTLSCipher` enum.  Would like to get your thoughts on this route.